### PR TITLE
add multiple-case

### PR DESCRIPTION
**Pull request by Chyroc**

### CODE REVIEW

**Comment by Chyroc** [comment]
fix https://github.com/mattn/anko/issues/189

File: vm/vmIfAndSwitch_test.go [hunk 1]
```diff
@@ -54,6 +54,29 @@ func TestSwitch(t *testing.T) {
 	os.Setenv("ANKO_DEBUG", "1")
 	tests := []testStruct{
 		// TODO: add tests
+
+		{script: `switch {}`, parseError: fmt.Errorf("syntax error")},
+		{script: `a=1; switch a; {}`, parseError: fmt.Errorf("syntax error")},
+
+		{script: `a=1; switch a {}`, runOutput: int64(1)},
+
+		{script: `a=1; switch a {case 1: return a}`, runOutput: int64(1)},
```
Comment (mattn) [inline vm/vmIfAndSwitch_test.go]: please add test for `switch a { case: return a}`

Comment (Chyroc) [reply vm/vmIfAndSwitch_test.go]: ok, and add 
```go
{script: `a=1; switch a {case: return a}`, parseError: fmt.Errorf("syntax error")},

{script: `a=99; switch a {default: return 'default'}`, runOutput: "default"},
```

**Comment by mattn** [comment]
Thank you

**Pull request merged**